provision-contest/ansible/emergency.yml

```diff
@@ -0,0 +1,57 @@
+---
+- hosts: domserver
+  tasks:
+    - name: Disable webserver to stop submissions
+      tags: powerloss
+      service:
+        nginx:
+          state: stopped
+
+- hosts: all
+  tasks:
+    - name: Disable GDM autologin
+      tags: full_lockdown
+      lineinfile:
+        path: /etc/gdm3/custom.conf
+        regexp: 'AutomaticLoginEnable'
+        line: 'AutomaticLoginEnable=false'
+        create: true
+        mode: 0644
+      when: GRAPHICAL
+
+- hosts: judgehost
+  tasks:
+    - name: Disable all running judgedaemons
+      tags: full_lockdown
+      service:
+        name: "domjudge-judgedaemon@{{ item }}"
+        state: stopped
+      with_sequence: start=0 end={{ ansible_processor_vcpus }} format=domjudge-judgedaemon@%1x
```

Comment:
Isn't this `format` duplicate with the service name already?

Reply:
Its copied from the judgehost.yml so if its wrong here its also wrong there.

Reply:
Fixed (IMO)